content/docs/introduction/getting_started.md

```diff
@@ -266,7 +266,32 @@ global: {
   rule_file: "prometheus.rules"
 }
 
-[...]
+job: {
```

Comment:
:+1: except that the sentence "The global section should now look like this" should now read "The config should now look like this"